tox.ini

```diff
@@ -4,6 +4,6 @@ envlist =
 
 [testenv]
 commands =
-    zope-testrunner --test-path=src
+    zope-testrunner --test-path=src []
```

Comment:
The  `[]` is a placeholder that tox fills in with additional arguments it gets passed. So you can do `tox -e py27 -- -t test_copy_then_delete_with_unicode_name` to debug just that one test in one environment.